### PR TITLE
include debug

### PR DESCRIPTION
**Pull request by AlexanderIbarra**
image

### CODE REVIEW

File: cypress/page/address-step.page.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export class Address {
+  private checkoutSelector: string;
```
Comment (mateoescobarm) [inline cypress/page/address-step.page.ts:2]: the element should be called after what it is, a button in this case "checkoutButtonSelector" for example

File: cypress/page/menu-content.page.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export class MenuContentPage {
+  private tShirtMenuSelector: string;
```
Comment (mateoescobarm) [inline cypress/page/menu-content.page.ts:2]: same

**Review by mateoescobarm — APPROVED** [review]
LGTM

**Comment by KevinIbarra-KSO** [comment]
:D

**Comment by KevinIbarra-KSO** [comment]
:D

**Pull request merged**